SUMMARY.md

```diff
@@ -92,7 +92,7 @@
     * SetPointColor
     * [DrawPoint](api-native-functions/drawpoint.md)
     * [DrawSide](api-native-functions/drawside.md)
-    * [SetRgbColor(r,g,b)](api-native-functions/setrgbcolorrgb.md)
+    * [SetRgbColor\(r,g,b\)](api-native-functions/setrgbcolorrgb.md)
```

Comment:
I’ve been changing these as I go - markdown (specifically, git-flavoured) doesn’t need to escape parentheses. The brackets will display without the \.
I can change this one, but it will save you putting them back in after I take them out! :)

Reply:
This is from the time when I was generating docs from futurocube.inc into LaTex. It is outdated and not needed. Thank you, Honza

Reply:
I just have found out, that GitBookEditor makes these as well... it escapes \_ and \(. So for the APIs we will have to check later, but originally it happen as I have described and needs to be corrected.